src/api/routes.py

```
@@ -0,0 +1,141 @@
+
+
+class AppConfig(BaseModel):
+    auth_enabled: Optional[bool] = False
+    host: Optional[str] = "127.0.0.1"
+    port: Optional[int] = 7788
+    theme: Optional[str] = "Ocean"
+
+
+# Routes
+@router.get("/browser/config", response_model=BrowserConfig)
+async def get_browser_config(current_user: User = Depends(get_current_active_user)):
```

Ruff (0.11.9)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)